system-tests/_scripts/launch-cluster.sh

```diff
@@ -49,8 +49,18 @@ then
   exit 1
 fi
 
-echo "http://$(dcos-launch describe -i ${CLUSTER_INFO} | python -c \
+URL="http://$(dcos-launch describe -i ${CLUSTER_INFO} | python -c \
                 'import sys, json; \
                  contents = json.load(sys.stdin); \
-                 sys.stdout.write(str(contents["masters"][0]["public_ip"]))')"
+                 print(contents["masters"][0]["public_ip"], end="")')"
+
+BACKOFF=1
+
```

Comment:
N00b question incoming: Isn't there a certain time frame that we have a confidence in that the cluster will never spawn that fast? E.g. 0s - 3 minutes or something along the lines. If we'd sleep that initially the exponential backoff times might not have gotten so big and we might actual ping a living cluster faster. Does that make sense?

Reply:
We don't have such statistics, we could come up with a number though 👍

Reply:
AFAIK this script already waits for `dcos-launch` to finish the cluster 🤔 

The thing is: The cluster might be online, but loading the UI right after the cluster is set up gives us a `connection refused` error which kills our proxy (`http-server`)…